ecommerce/extensions/checkout/views.py

```diff
@@ -194,8 +194,30 @@ def get_context_data(self, **kwargs):  # pylint: disable=arguments-differ
             'has_enrollment_code_product': has_enrollment_code_product,
             'disable_back_button': self.request.GET.get('disable_back_button', 0),
         })
+        user = order.user
```

Comment:
@alfredchavez, can you remove the code already moved to the `ecommerce-paytabs` pluggable app before we can merge this?

Reply:
ok